common/services/prismic/opening-times.ts

```diff
@@ -248,21 +248,21 @@ type ExceptionalOpeningHoursGroup = ExceptionalOpeningHoursDay[];
  */
 export function createExceptionalOpeningHoursDays(
   venue: Venue,
-  allVenueExceptionalPeriods?: ExceptionalPeriod[]
+  allVenueExceptionalPeriods: ExceptionalPeriod[]
 ): ExceptionalOpeningHoursGroup[] {
   const groupedExceptionalDays = groupExceptionalVenueDays(
     venue.openingHours.exceptional
   );
-  return (allVenueExceptionalPeriods ?? []).map(period => {
+  return allVenueExceptionalPeriods.map(period => {
```

Comment:
In the one place we call this function, `allVenueExceptionalPeriods` is defined, so we can just go ahead and skip the optional type here.